addons/dragging/functions/fnc_handleScrollWheel.sqf

```diff
@@ -30,16 +30,16 @@ private _carriedItem = _unit getVariable [QGVAR(carriedObject), objNull];
 //disabled for persons
 if (_carriedItem isKindOf "CAManBase") exitWith {false};
 
-private _position = getPosATL _carriedItem;
-private _maxHeight = (_unit modelToWorldVisual [0,0,0]) select 2;
+private _position = getPosASL _carriedItem;
+private _maxHeight = (_unit modelToWorldVisualWorld [0, 0, 0]) select 2;
 
 _position set [2, ((_position select 2) + _scrollAmount min (_maxHeight + 1.5)) max _maxHeight];
 
 // move up/down object and reattach at current position
 detach _carriedItem;
 
 // Uses this method of selecting position because setPosATL did not have immediate effect
-private _positionChange = _position vectorDiff (getPosATL _carriedItem);
+private _positionChange = _position vectorDiff (getPosASL _carriedItem);
 private _selectionPosition = _unit worldToModel (ASLtoAGL getPosWorld _carriedItem);
```

Comment:
modelToWorldVisualWorld ?

Reply:
That would probably work too. I just converted the AGL stuff to ASL to get this out asap.

Reply:
Also you can reuse `_position` instead of calling getPos again. Which is relativly expensive. Not as expensive as getPosATL tho.. I think.

Reply:
Can do for future version in separate PR.